src/OpenTracing/Extractor.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace OpenTracing;
+
+/**
+ * An Extractor extracts Span instances from a format-specific "carrier" object.
+ *
+ * Typically the carrier has just arrived across a process boundary, often via
+ * an RPC (though message queues and other IPC mechanisms are also reasonable
+ * places to use an Extractor).
+ *
+ * See Injector and Tracer.extractor().
+ *
+ * @package OpenTracing
+ */
+interface Extractor {
```

Comment:
Modern PHP follows the Java-like practise to put each class/interface in a separate file so I think this should stay as it is.